docs/contributing.md

```diff
@@ -35,25 +35,27 @@ Our [Go release workflow](https://github.com/treeverse/lakeFS/blob/master/.githu
 
 1. Install the required dependencies for your OS:
    1. [Git](https://git-scm.com/downloads)
-   1. [GNU make](https://www.gnu.org/software/make/) (probably best to install from your OS package manager such as apt or brew)
+   1. [GNU make](https://www.gnu.org/software/make/) (probably best to install from your OS package manager such as [apt](https://en.wikipedia.org/wiki/APT_(software)) or [brew](https://brew.sh/))
    1. [Docker](https://docs.docker.com/get-docker/)
    1. [Go](https://golang.org/doc/install)
    1. [Node.js & npm](https://www.npmjs.com/get-npm)
    1. [Maven](https://maven.apache.org/) to build and test Spark client codes.
+   1. Java 8, which you can install from [Azul Zulu Builds for Java JDK](https://www.azul.com/downloads/?package=jdk)
```

Comment:
Why recommend Azul for non-Mac users?  E.g. [Fedora has current support for Java 8](https://docs.fedoraproject.org/en-US/quick-docs/installing-java/).  Linux users (let alone GNU/Linux users...) might not like Azul as much; there is no requirement for this particular vendor and I'd rather they got a package with an easier upgrade path.

Reply:
Ah good point - it was just the Apple M1 comment that linked to this. I'll rephrase.